------------------------------------------------------------
revno: 14177
revision-id: squid3@treenet.co.nz-20170701073754-4x1i6p5s1gzk73co
parent: squid3@treenet.co.nz-20170701073514-uzowexcysowqostf
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Sat 2017-07-01 19:37:54 +1200
message:
  basic_ncsa_auth: fix hash listing wrap in man(8) page
  
  '*' list bullet points must be indented with whitespace.
  If they are not the list is treated as a single wrapped paragraph.
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170701073754-4x1i6p5s1gzk73co
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: ffd783ab19a438c56affcdc6c1d106fa00403f4f
# timestamp: 2017-07-01 07:45:09 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170701073514-\
#   uzowexcysowqostf
# 
# Begin patch
=== modified file 'helpers/basic_auth/NCSA/basic_ncsa_auth.8'
--- helpers/basic_auth/NCSA/basic_ncsa_auth.8	2017-01-01 00:16:45 +0000
+++ helpers/basic_auth/NCSA/basic_ncsa_auth.8	2017-07-01 07:37:54 +0000
@@ -18,15 +18,15 @@
 .PP
 This authenticator accepts:
 .BR
-* Blowfish - for passwords 72 characters or less in length
-.BR
-* SHA256 - with salting and magic strings
-.BR
-* SHA512 - with salting and magic strings
-.BR
-* MD5 - with optional salt and magic strings
-.BR
-* DES - for passwords 8 characters or less in length
+ * Blowfish \- for passwords 72 characters or less in length.
+.BR
+ * SHA256 \- with salting and magic strings.
+.BR
+ * SHA512 \- with salting and magic strings.
+.BR
+ * MD5 \- with optional salt and magic strings.
+.BR
+ * DES \- for passwords 8 characters or less in length.
 .
 NOTE: Blowfish and SHA algorithms require system-specific support.
 .

